Documentation/Grafana-Dashboards/ContainerLogV2.json

```diff
@@ -0,0 +1,2533 @@
+{
+    "annotations": {
+      "list": [
+        {
+          "builtIn": 1,
+          "datasource": {
+            "type": "grafana",
+            "uid": "-- Grafana --"
+          },
```

Comment:
Add new bar chart for Log Records by Pod

Reply:
added